src/libs/getIsNarrowLayout/index.native.ts

```diff
@@ -0,0 +1,3 @@
+export default function getIsNarrowLayout() {
+    return true;
```

Comment:
This is an unsafe assumption – there are devices (such as an iPad pro), where even in portrait layout the screen width is wider than the breakpoint for narrow screens.

I think this practice of using platform as a proxy for other device properties which can be measured directly is bad and a source for technical debt.

I feel strongly that this PR be closed and we fix this by addressing the root cause of the problem(s) on iPad. My hunch is that most of them are stemming from bad assumptions like this one that native code _must_ be running on a device with narrow screen width.

Reply:
I feel like this is more of a feature request as the assumption of us forcing portrait mode and narrow layout is running for a while and until we decide to officially support landscape too, its fine to keep it in the app.

Reply:
> bad assumptions like this one that native code must be running on a device with narrow screen width.

This is not bad assumption imho, this is a intentional product/ design choice made in the past

Reply:
> even if the screen width is beyond the threshold, the iPad always displays narrow layout

We already applied this throughout the app so I think it's fine for now.
Agree we should completely refactor this bad logic when we support wide layout in portrait mode of super large iPad (maybe when we officially support landscape mode)